test-it/ExpectedResults/JSReferenceArraySearch.vtr

```diff
@@ -0,0 +1,3 @@
+dataItem_IndexOfElement: 2
+dataItem_TestResult_CorrectIndex: true
+dataItem_TestResult_NoError: true
```

Comment:
Why even calculate and print this? dataItem_TestResult_NoError

Reply:
We need to know that there were any errors with JSLI calls.

Reply:
You don't need to fix it now, as we have not had this process around this yet. I think it is better for us to start thinking about it together. It would be better to distinguish the "core asserts" that the the test is intending to assert vs some of the "pre-conditions that we want to assert". May be, we should prefix/suffix the pre-condition asserts with "_PreCondtion_".